packages/adp-tooling/README.md

```diff
@@ -1,12 +1,23 @@
 #  `@sap-ux/adp-tooling`
 
-Module containing different tooling modules helpful when working with SAP UI5 adaptation projects.
+A module containing different tooling modules helpful when working with SAP UI5 adaptation projects.
 
-## preview
+## Submodules
+
+### preview
 The submodule preview contains functionality allowing to preview adaptation projects. It is not a standalone UI5 middleware but designed to be integrated into the `@sap-ux/preview-middleware.`.
 
-## writer
+#### preview/client
+This is a special module that is transpiled separately to run in the browser as part of the UI5 RTA overlay. Any communication with the middleware middleware (server) code is done via http.
+
+### writer
 The submodule writer contains functionality to generate the core project structure of an SAP UI5 adaptation project. It is not a standalone generator but designed to be integrated into `@sap-ux/create` or any kind of yeoman generator.
 
-## base
-The submodule contains functionality required in different scenarios, e.g. prompting for generation or when initializing the preview.
\ No newline at end of file
+### base
+The submodule contains functionality required in different scenarios, e.g. prompting for generation or when initializing the preview.
+
+## Development
+The standard `src` folder except the subfolder `src/preview/client` is transpiled into the dist folder. Additionally, the sources from `src/preview/client` are transpiled using babel into UI5 compatible client sources that can be used in previews to extend the runtime adaption with additional plugins. 
```

Comment:
I'd like to wait for the outcome of #1231. If the decision is to keep a flat structure, I'd really prefer to keep it together because it just confusing to have the client module standalone as a top level package. If we go for nested, then I am more than happy to split it apart.

Reply:
@devinea would you be ok with https://github.com/SAP/open-ux-tools/compare/feat/1149/enable-enhancements...feat/1149/enable-enhancements-module?expand=1? I am moving the client source to a new module called `rta-client-extension`.
The ongoing work on the `control-properties-editor` will then also enhance that module and use it.

Reply:
@tobiasqueck yes that is better thanks 👍

Reply:
This is obsolete with the latest changes in `main`.